openood/evaluators/ood_evaluator.py

```diff
@@ -25,34 +25,56 @@ def __init__(self, config: Config):
         self.id_conf = None
         self.id_gt = None
 
-    def eval_ood(self, net: nn.Module, id_data_loader: DataLoader,
+    def eval_ood(self,
+                 net: nn.Module,
+                 id_data_loaders: Dict[str, DataLoader],
                  ood_data_loaders: Dict[str, Dict[str, DataLoader]],
-                 postprocessor: BasePostprocessor):
+                 postprocessor: BasePostprocessor,
+                 fsood: bool = False):
         if type(net) is dict:
             for subnet in net.values():
                 subnet.eval()
         else:
             net.eval()
-        # load training in-distribution data
-        assert 'test' in id_data_loader, \
+        assert 'test' in id_data_loaders, \
             'id_data_loaders should have the key: test!'
         dataset_name = self.config.dataset.name
+
+        if self.config.postprocessor.APS_mode:
```

Comment:
@ALin

Reply:
Sounds good. Thank you for your work!

Reply:
It looks like in the wiki doc, the leaderboard webpage is not on yet. Any plans for releasing that? That would be really helpful for the community.

Reply:
A preview can be found [here](https://zjysteven.github.io/OpenOOD/).

Reply:
Awesome! Thank you so much!

Reply:
One more question, I think the Tiny ImageNet validation datasets .txt file is not available on one drive right now. Can you share that as well? Thanks!

Reply:
The download link to `benchmark_imglist` was updated, and the updated version includes all the new or changed .txt files. Please take a look.
https://github.com/Jingkang50/OpenOOD/blob/f8870d72e3d0abb0c93c123fed7a2d2800f54c2a/scripts/download/download.py#L71